### DISCUSSION TITLE
Add the function of set or update table optimizer groups through ams web ui.

### DISCUSSION THREAD

**Discussion by MrDannyWu** [apache/amoro#2129] [Ideas]
During my use, I need to set up different groups to optimize different data tables. I need to manually modify the table properties every time, which is a bit cumbersome to operate. And it's easy to forget, resulting in using the default grouping. If the default grouping is not enabled or I do not want to use the default grouping, this table will not be merged. Or enter the default group, and the pressure of the default group increases, eventually causing some impact.
Therefore, I am thinking about whether I can set or update the optimizer group of the table through ams web ui, and display which tables under each group are being optimized.
I hope we can discuss the specific layout and design of this function, thank you all!

**Comment by wangtaohz** [comment]
Thank you for starting this discussion, and this issue has also caused me some inconvenience.

I think it is valuable to support modifying optimizer group on Web UI, rather than executing SQL to modify it through the `Terminal`. My personal suggestion is to support modifying it on the `Optimizing/Tables` page, just like:

<img width="1878" alt="image" src="https://github.com/NetEase/amoro/assets/103108928/ef7a0484-be65-42b4-8889-84b92626cb4b">


As for **displaying which tables under each group are being optimized** that you mentioned, I think the `Optimizing/Tables` page has already achieved this purpose, but it would be more convenient to use if this page could support filtering optimizer group and searching with table name.

**Comment by shidayang** [comment]
Perhaps we can extend the following page by adding a table to display the list and a button to add a table.
![image](https://github.com/NetEase/amoro/assets/26699250/02fd1c7e-4ac0-4bfc-adc2-08ea748f33d7)

**Comment by majin1102** [comment]
Thanks for sharing you idea, it's important for Amoro community.

I raised a first good issue to solve this scenario: https://github.com/NetEase/amoro/issues/1865. I suggest you could make your contribution in this issue and I would love to provide any help you need
